doc/release-notes/release-6.sgml

```diff
@@ -25,7 +25,14 @@ This new release is available for download from <url url="http://www.squid-cache
 <sect1>Known issues
 <p>Although this release is deemed good enough for use in many setups, please note the existence of
 <url url="http://bugs.squid-cache.org/buglist.cgi?query_format=advanced&amp;product=Squid&amp;bug_status=UNCONFIRMED&amp;bug_status=NEW&amp;bug_status=ASSIGNED&amp;bug_status=REOPENED&amp;version=6" name="open bugs against Squid-6">.
-<p>Support for compiling on HPUX with the native HP <em>xcc</em> compiler has been removed. To build on that OS/compiler combination, it is possible to pass these environemnt variables to ./configure: <em>CC="cxx -Ae" RANLIB=":"</em>
+
+<p>Support for compiling on HPUX with the native HP <em>xcc</em> compiler has been removed.
+  To build on that OS/compiler combination, it is possible to pass these environment variables
+  to ./configure: <em>CC="cxx -Ae" RANLIB=":"</em>
+
+<p>This release adds a dependency on C++17 support in any compiler used to build Squid.
+  GCC 8+ and Clang 8+ are known to have working C++17 support and are usable.
```

Comment:
Do we know for sure that GCC v8 can build this current Squid code? I have not tested that. If we are not sure, I recommend relaxing the above claim:

```suggestion
  GCC 8+ and Clang 8+ support C++17.
```